src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,330 @@
+package duke;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+import duke.exception.DukeCommandDoesNotExistException;
+import duke.exception.DukeException;
+import duke.io.Parser;
+import duke.io.UserInput;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+/**
+ * Duke is a task tracker interactive chatbot.
+ *
+ * @author Chen Yu An
```

Comment:
I liked how you included the author and version number here!

Reply:
Thank you 😄